src/index.html

```diff
@@ -11,7 +11,167 @@
+            name="name"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input
+            type="number"
+            autocomplete="off"
+            value="12"
+            min="1"
+            max="100"
+            name="age"
+          >
+        </label>
+
+        <label class="form-field">
+          Full date of birth:
+          <input
+            type="date"
+            name="birthDate"
+          >
+        </label>
+
+        <label class="form-field, field" required>
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            name="termAccepted"
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset class="section">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            autocomplete="off"
+            required
+            name="eMail"
+          >
+        </label>
+
+        <label class="form-field, field">
+          Password:
+          <input
+            type="password"
+            autocomplete="off"
+            minlength="5"
+            maxlength="32"
+            required
+            name="password"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="section">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label>
+            <input type="radio" name="loveCats">
+              Yes
+          </label>
+          <label>
+            <input type="radio" name="loveCats">
+              No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input
+            type="color"
+            name="likesColor"
+          >
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input
+            type="time"
+            name="goesToBed"
+          >
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+          <select name="likesCar" multiple>
+            <option value="1">BMW</option>
+            <option value="2">Audi</option>
+            <option value="3">Lada</option>
+            <option value=" " disabled></option>
+          </select>
+        </label>
+
+        <label class="form-field, field">
+          How do you rate our work?
+          <input
+            type="range"
+            name="workRate"
+            min="1"
+            max="10"
+            value="1"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="section">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea
+            name="comments"
+            cols="20"
+            rows="2">
+          </textarea>
```

Comment:
fix formatting here, your text area works weird cause it uses new line in your html code like a value:
<img width="294" alt="image" src="https://user-images.githubusercontent.com/37161315/165489735-b2cef8cf-d539-4cbe-a7ae-49dc589e06e6.png">
```suggestion
          <textarea
            name="comments"
            cols="20"
            rows="2"
         ></textarea>
```